src/include/migraphx/generate.hpp

```diff
@@ -82,6 +82,8 @@ std::vector<T> generate_tensor_data(const migraphx::shape& s, unsigned long seed
 {
     std::vector<T> result(s.elements());
     std::generate(result.begin(), result.end(), xorshf96_generator<T>{seed});
+    // divide a value to avoid integer overflow
+    std::transform(result.begin(), result.end(), result.begin(), [](auto i) { return i / 32; });
```

Comment:
The max values on signed integers are 32, so this will make everything a 1 or 0. I dont think we want that.

Reply:
And changes such as this, should be done in the `normalize` function.

Reply:
We already did that in the normalize function, so just remove it here.